zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1893,6 +1933,77 @@ components:
           minimum: 0
           maximum: 100
           default: 50
+    ProcessDefinitionSearchQueryRequest:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      type: object
+      properties:
+        filter:
+          allOf:
+            - $ref: "#/components/schemas/ProcessDefinitionFilterRequest"
+    ProcessDefinitionFilterRequest:
+      type: object
+      properties:
+        processDefinitionKey:
+          description: The key for this process definition.
+          type: integer
+          format: int64
+        processName:
+          description: Name of this process definition.
+          type: string
+        processVersion:
+          description: Version of this process definition.
+          type: integer
+          format: int32
+        processVersionTag:
+          description: Version tag of this process definition.
+          type: string
+        bpmnProcessId:
+          description: BPMN process id of this process definition.
+          type: string
+        tenantId:
+          description: Tenant id of this process definition.
+          type: string
+        formKey:
+          description: The key of the form related to this process definition.
+          type: integer
+          format: int64
+    ProcessDefinitionSearchQueryResponse:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryResponse"
+      type: object
+      properties:
+        items:
+          type: array
+          items:
+            $ref: "#/components/schemas/ProcessDefinitionItem"
+    ProcessDefinitionItem:
```

Comment:
There is an extension property over web modeler, where set when a process is public (which can be started without authentication). 

I think this should be considered on the operate process index update, as well as the return on the item. 

It can be done in a separate pull request.

See as reference: https://docs.google.com/document/d/1gKV8WgpGZuCUyu7yyx2wFNSaRtRUDzoPgKn2NaA-i4s/edit

We dont need cover all requisites here, but it can be aligned with modeler to have the property as we do on tasklist-process when the process is public or not.

Reply:
Will be addressed in follow up PR. Thanks for raising this!